doc/multiplier.md

```diff
@@ -0,0 +1,24 @@
+# Multiplier
+
+ROHD HCL provides a multiplier module to get the product from a list of logic. As of now, we have
+
+- [Carry Save Multiplier](#carry-save-multiplier)
+
+## Carry Save Multiplier
+
+Carry save multiplier is a digital circuit used for performing multiplication operations. It is particularly useful in applications that require high speed multip0lication, such as digital signal processing.
+
+The [`CarrySaveMultiplier`](https://intel.github.io/rohd-hcl/rohd_hcl/CarrySaveMultiplier-class.html) module in ROHD-HCl accept four inputs: the clock `clk` signal, reset `reset` signal, a List of Logic()'s `toMultiply` and the name of the module `name`. Note that `toMultiply` must have inputs of two and width of the inputs must be the same or `RohdHclException()` will be thrown.
```

Comment:
Instead of "Logic()", say `Logic` (in ticks).  This applies in the adder doc also.  There's no need to add `()` after class names (e.g. in `RohdHclException()`)